content/400-reference/300-database-reference/01-database-features.mdx

```diff
@@ -59,8 +59,10 @@ This section describes which database features exist on the relational databases
 | Index    | PostgreSQL | Microsoft SQL Server | MySQL | SQLite | CockroachDB | Prisma schema                                                                                                                | Prisma Client | Prisma Migrate |
 | -------- | ---------- | -------------------- | ----- | ------ | ----------- | ---------------------------------------------------------------------------------------------------------------------------- | ------------- | -------------- |
 | `UNIQUE` | ✔️         | ✔️                   | ✔️    | ✔️     | ✔️          | [`@unique` and `@@unique`](/concepts/components/prisma-schema/data-model#defining-a-unique-field)                            | ✔️            | ✔️             |
+| `USING`  | ✔️         | No                   | No    | No     | ✔️          | [`type`](/concepts/components/prisma-schema/indexes#configuring-the-access-type-of-indexes-with-type-postgresql)<sup>†</sup> | ✔️            | ✔️<sup>†</sup> |
 | `WHERE`  | ✔️         | ✔️                   | ✔️    | ✔️     | ✔️          | Not yet                                                                                                                      | ✔️            | Not yet        |
-| `USING`  | ✔️         | ✔️                   | ✔️    | ✔️     | ✔️          | [`type`](/concepts/components/prisma-schema/indexes#configuring-the-access-type-of-indexes-with-type-postgresql)<sup>†</sup> | ✔️            | Not yet        |
+| `(expression)` | ✔️         | ✔️                   | ✔️    | ✔️     | ✔️         | Not yet                                                                                                                   | ✔️            | Not yet        |
```

Comment:
Does this refer to something like https://www.postgresql.org/docs/current/indexes-expressional.html?

If so maybe 'Indexes on expressions' would be clearer?

Reply:
Yes, I think that is what I meant back then. They are just called "Expression indexes" as well sometimes, but in the SQL they are marked by being in `( ... )` - that might have been my thought back in November.